designs/0016-hessian_optimize_cmdstan.md

```diff
@@ -13,7 +13,16 @@ When computing a MAP estimate, the Hessian of the log density can be used to con
 
 I have a Stan model that is too slow to practically sample all the time. Because optimization seem to give reasonable results, it would be nice to have the normal approximation to the posterior to give some sense of the uncertainty in the problem as well.
 
-An approximate posterior covariance comes from computing the inverse of the Hessian of the negative log density.
+It is standard to compute a normal approximation to the posterior covariance comes from the inverse of the Hessian of the negative log density.
```

Comment:
remove "comes" (I would remove it myself, but I don't have edit rights. It would be useful to have edit rights for design-docs repo)